src/components/SubmissionForm/SubmissionForm.js

```diff
@@ -44,14 +52,22 @@ const SubmissionForm = () => {
       return 
     } else if (event.target.files[0].size >= 500000000) {
       setIsLargeFile(true)
+    } else if (!event.target.files[0].type.includes('image')) {
+      setIsNotImageFile(true)
     } else {
       setIsLargeFile(false)
+      setIsNotImageFile(false)
       setProfileImage(event.target.files[0])
     }
   }
 
+  useEffect(() => {
+    return () => {setIsUploading(false)}
```

Comment:
Not sure if having this return a callback is necessary or if we can just call setIsUploading(false) inside of the useEffect hook. I think it's fine to leave for now if it's working - but is something we can look into when refactoring!